app/views/furnitures/my_furnitures.html.erb

```diff
@@ -0,0 +1,12 @@
+<ul>
```

Comment:
It would probably be nicer to make cards instead of a list
<img width="1427" alt="Screenshot 2022-02-22 at 14 11 07" src="https://user-images.githubusercontent.com/66831769/155149296-9b5e55d2-1922-4cdc-8efa-7cc627b0fa82.png">
t